src/uu/stat/src/stat.rs

```diff
@@ -26,67 +26,63 @@ use std::os::unix::prelude::OsStrExt;
 use std::path::Path;
 use std::{cmp, fs, iter};
 
-macro_rules! check_bound {
-    ($str: ident, $bound:expr, $beg: expr, $end: expr) => {
-        if $end >= $bound {
-            return Err(USimpleError::new(
-                1,
-                format!("{}: invalid directive", $str[$beg..$end].quote()),
-            ));
-        }
-    };
+fn check_bound(slice: &str, bound: usize, beg: usize, end: usize) -> UResult<()> {
```

Comment:
it would be nice if you could do document this function (and the following)
https://doc.rust-lang.org/book/ch14-02-publishing-to-crates-io.html#commenting-contained-items

Reply:
sure, will do.